code-gen-projects/schema/java/enum.isl

```diff
@@ -0,0 +1,4 @@
+type::{
+  name: enum,
+  valid_values: ["foo", bar, "baz"]
```

Comment:
Hmm. Thats true, it will remove duplicates since I am using `Hashset`. 
Did you mean require all the values to be of same type either symbols or strings?
e.g.
```
valid_Values: [foo, bar, baz]
// or
valid_values: ["foo", "bar", "baz"]
```

Reply:
No, I mean just require them to all be symbols. It simplifies things, and in terms of best practices, an enum value is a great candidate for something that goes in the symbol table because it has a reasonably high likelihood of recurring in multiple places.

Reply:
Made changes to only allow symbols for `valid_values`